Fix broken links in our error docs (#1136)
ngrok/ngrok-docs: docs/errors/details/_err_ngrok_306.md

### Possible Solutions

> - You can upgrade to a paid plan in [the Billing section of the dashboard](https://dashboard.ngrok.com/billing). You will need a Personal, Pro or Enterprise plan to use a custom hostname.

Reviewer: ```suggestion
- You can upgrade to a paid plan in [the Billing section of the dashboard](https://dashboard.ngrok.com/billing). You will need at least a Personal plan or any pay-as-you-go-plan.
```
